pygments/lexers/scripting.py

```diff
@@ -163,6 +163,321 @@ def get_tokens_unprocessed(self, text):
                     continue
             yield index, token, value
 
+def _luau_make_expression(should_pop, _s):
```

Comment:
Can you please add a comment as to what this function does and rename `_s` to something more readable?